src/config/unit_test/mod.rs

```diff
@@ -292,15 +290,14 @@ impl UnitTestBuildMetadata {
             };
 
             test_result_rxs.push(rx);
-            template_sinks.insert(id.clone().into(), sink_config);
+            template_sinks.insert(vec![id.clone()], sink_config);
```

Comment:
Is this a stylistic change, or is there a benefit to it?

Reply:
Not stylistic. I made `template_sinks` use `Vec<OutputId>` instead of `OneOrMany<OutputId>` for the `K` in `HashMap<K, V>`. There's no `impl<T> From<T> for Vec<T>` which is what `OneOrMany<T>` has, so it needs to be wrapped in a `Vec<T>` explicitly now.